Sources/TokamakStaticHTML/Core.swift

```diff
@@ -55,6 +76,8 @@ public typealias CGSize = TokamakCore.CGSize
 
 // MARK: Views
 
+public typealias Button = TokamakCore.Button
```

Comment:
I intentionally left out things like `Button` and `TextField` as they're pretty much useless without `State`. Do you think we should include them?

Reply:
What we can add which will make TokamakStaticHTML more useful is [`Link`](https://developer.apple.com/documentation/swiftui/link)

Reply:
I wonder if we could still render stateful `View`s to static HTML, any listeners on those would be just ignored. But that would obviously come as a separate PR. LMK if you think that these should be removed in this PR.

Reply:
I think they should be removed, plus they only have HTML bodies in TokamakDOM.